.github/workflows/validate.yml

```diff
@@ -29,3 +29,7 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: ./build.sh
+      - uses: JustinBeckwith/linkinator-action@v1
+        with:
+          paths: 'dist'
```

Comment:
Only checking the Astro site here. I think this is ok as the Jekyll site will be removed soon 💅